### PR TITLE
Fixes version numbers in katello

### PR DESCRIPTION
**Pull request by johnpmitsch**
Many links are broken because we aren't using the variables from the `page` instead of the `site`

### CODE REVIEW

File: plugins/katello/2.4/cli/index.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 layout: plugins/katello/documentation
 title: CLI
 version: 3.2
+foreman_version: 1.13
```
Comment (ehelms) [inline plugins/katello/2.4/cli/index.md]: This should be 2.4 and 1.9 for the katello and foreman versions respectfully.

Comment (ehelms) [reply plugins/katello/2.4/cli/index.md]: Err sorry, 1.10 of Foreman

File: plugins/katello/2.4/installation/index.md [hunk 2]
```diff
@@ -2,8 +2,9 @@
 layout: plugins/katello/documentation
 title: Katello Installation
 version: 2.4
-foreman_version: 1.10
+foreman_version: "1.10"
```
Comment (ehelms) [inline plugins/katello/2.4/installation/index.md:5]: Why does this change end up being necessary?

File: plugins/katello/3.1/cli/index.md [hunk 3]
```diff
@@ -2,6 +2,7 @@
 layout: plugins/katello/documentation
 title: CLI
 version: 3.2
+foreman_version: 1.13
```
Comment (ehelms) [inline plugins/katello/3.1/cli/index.md]: Looks like this should be 3.1 and 1.12

**Review by ehelms — CHANGES_REQUESTED** [review]

File: plugins/katello/2.4/installation/index.md (hunk 2 above)
Comment (johnpmitsch) [reply plugins/katello/2.4/installation/index.md:5]: it will show as `1.1` where its used in the page

**Comment by johnpmitsch** [comment]
@ehelms updated

**Review by ehelms — APPROVED** [review]

**Comment by ehelms** [comment]
@johnpmitsch looks good! Maybe @stbenjam can merge this update for us

**Comment by stbenjam** [comment]
Looks good, thanks!

**Pull request merged**